package.json

```diff
@@ -7,7 +7,6 @@
         "@fortawesome/fontawesome-svg-core": "^1.2.32",
         "@fortawesome/free-brands-svg-icons": "^5.15.1",
         "@fortawesome/react-fontawesome": "^0.1.12",
-        "@ant-design/icons": "^4.2.2",
```

Comment:
is this intended to be deleted?

Reply:
It is not comletely deleted, version of the icons changed after latest merge.